appinventor/appengine/src/com/google/appinventor/client/explorer/youngandroid/AssetList.java

```diff
@@ -95,15 +103,38 @@ public void onMouseMove(MouseMoveEvent event) {
         clientY = event.getClientY();
       }
     });
+    
+    
     assetList.addSelectionHandler(new SelectionHandler<TreeItem>() {
+      private int clickCount = 0;
       @Override
-      public void onSelection(SelectionEvent<TreeItem> event) {
-        TreeItem selected = event.getSelectedItem();
-        ProjectNode node = (ProjectNode) selected.getUserObject();
-        // The actual menu is determined by what is registered for the filenode
-        // type in CommandRegistry.java
-        ProjectNodeContextMenu.show(node, selected.getWidget(), clientX, clientY);
-      }});
+      public void onSelection(final SelectionEvent<TreeItem> event) {
+         clickCount++;
+         if (clickCount == 1) {
+            Timer timer = new Timer() {
+            @Override
```

Comment:
Following indentation, this should be in two spaces from the previous line. For an example, see the initialization of autoSaveTimer in EditorManager.java.